packages/reports/src/Constants/data-table-constants.tsx

```diff
@@ -25,20 +37,43 @@ const getModeFromValue = key => {
         transfer: 'transfer',
     };
 
-    if (Object.keys(map).find(x => x === key)) {
-        return map[key];
-    }
+    return map[key] || map.default;
+};
+
+type TCellContentProps = {
+    cell_value: string;
+    passthrough: any;
+    row_obj: any;
```

Comment:
same for this

Reply:
The same concerns I have about row_obj, since it varies differently among components and should be extendable.